.gitignore

```diff
@@ -17,3 +17,22 @@
 configure.wrf*
 *.backup
 *.f90
+
+# Out-of-source build locations
+_build*
+wrf_config.cmake
+
+# Executables when not featuring .exe suffix
+ndown
+real
+tc
+ideal
+wrf
+
+# Model inputs/outputs
+wrfbdy_d*
+wrfinput_d*
+wrfout_d*
+*.nc
+rsl.out.*
+rsl.error.*
```

Comment:
The only change I'd suggest is to add an end-of-line character after the last line.